src/docfx/lib/UrlUtility.cs

```diff
@@ -344,6 +344,21 @@ public static string RemoveLeadingHostName(string url, string hostName, bool rem
             : $"/{path}";
     }
 
+    public static string GetBookmark(string uid)
+    {
+        uid = uid.ToLowerInvariant();
```

Comment:
would make sense to use `StringBuilder`
https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.replace?view=net-6.0#System_Text_StringBuilder_Replace_System_Char_System_Char_

Reply:
And we probably don't need these regular expressions either, they are overkill and slow.

Reply:
Fixed.